Lab3.ipynb

```diff
@@ -0,0 +1,366 @@
+   "metadata": {},
+   "source": [
+    "The statistical question I have is what is the displacement or movement to see a 5-sigma detection from the background in arcseconds? The reason this is my statistical question is because an asteroid will have some true movement between the images. "
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "**Section 2.2**"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "From the code, we have a 2D Gaussian distribution with 100k data points where mean is 0 and sigma is 1. We would want to find at what value in the distribution corresponds to a 5-sigma detection from the background. "
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "**Section 2.3**"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Here, we want to find the value which corresponds to 5-sigma detection from the background. Below is our calculation."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 7,
+   "metadata": {},
+   "outputs": [
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "5.49\n"
+     ]
+    }
+   ],
+   "source": [
+    "det = scipy.stats.rayleigh.ppf(stats.norm.cdf(5), loc=0, scale=1)\n",
+    "print(round(det, 2))"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "### **Problem 3**"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "At a trillion electron volt energies and above, the moon and the sun will block cosmic ray background. This means the sun and the moon will appear as holes in the cosmic ray sky since the rays are absorbed. We want to calculate the significance of our moon shadow detection. "
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "**Section 3.1**"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "We assume we normally have a cosmic ray rate of 1 ray per minute in a moon sized patch in the sky. We want to know the significance of our moon shadow detection if we see 6800 cosmic rays if we observe where the moon is for 8 hours per night for 15 days. Specifically, my question is what is the probability of the background producing a count of 6800 cosmic rays or less?"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "**Section 3.2**"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "The expected background pdf() is a Poisson distribution because we only see an integer number of cosmic rays. We want to be integrating from negative infinity to 6800 which corresponds to seeing 6800 cosmic rays or less. Specifically, this is the left tail of our distribution. This means the integral we would need to do is an integral of a Poisson distribution from negative infinity to 6800. "
```

Comment:
Poisson is zero for negative values since it is the probability of a count of events, so your integral can start at negative infinity but from negative infinity to zero you will get no contribution to the probability.  so can can totally start your integral at 0